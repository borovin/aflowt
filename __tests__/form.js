import test from 'ava'
import Browser from './helpers/browser'
import {prettyPrint} from 'html'

test.beforeEach(t => {
  t.context.browser = new Browser()

  return t.context.browser.goto('/examples/form.html')
})

test.afterEach(t => {
  return t.context.browser.close()
})

test('Form example', async t => {
  const snapshot = await t.context.browser.snapshot('b-form')

  t.snapshot(prettyPrint(snapshot))
})
