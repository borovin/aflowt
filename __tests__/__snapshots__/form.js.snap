// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form example 1`] = `
"<b-form action=\\"/examples/form.html\\">
    <form>
        <b-input name=\\"A\\" label=\\"A field\\" type=\\"text\\">
            <b-input-text name=\\"A\\" label=\\"A field\\" type=\\"text\\">
                <input type=\\"text\\" name=\\"A\\">
                <b-input-text--border></b-input-text--border>
                <b-input-text--label>A field</b-input-text--label>
            </b-input-text>
        </b-input>
        <b-input name=\\"B\\" label=\\"B field\\" value=\\"default B value\\" type=\\"text\\">
            <b-input-text name=\\"B\\" label=\\"B field\\" value=\\"default B value\\" type=\\"text\\">
                <input value=\\"default B value\\" type=\\"text\\" name=\\"B\\">
                <b-input-text--border></b-input-text--border>
                <b-input-text--label>B field</b-input-text--label>
            </b-input-text>
        </b-input>
        <b-button color=\\"primary\\">
            <button>Save</button>
        </b-button>
    </form>
</b-form>"
`;
